We are not interrested in coverage report (mainly because it does not work).

--- python-dotenv-1.0.0/tox.ini.orig
+++ python-dotenv-1.0.0/tox.ini
@@ -17,7 +17,7 @@
   sh >= 2.0.2, <3
   click
   py{38,39,310,311,py312-dev,pypy3}: ipython
-commands = pytest --cov --cov-report=term-missing --cov-config setup.cfg {posargs}
+commands = pytest {posargs}
 depends =
     py{38,39,310,311,312-dev},pypy3: coverage-clean
     coverage-report: py{38,39,310,311,312-dev},pypy3
